docs/source/blocks/core/grid.md

```diff
@@ -0,0 +1,148 @@
+---
+myst:
+  html_meta:
+    "description": "Volto Grid Block description and developer notes"
+    "property=og:description": "Volto Grid Block description and developer notes"
+    "property=og:title": "Volto Grid Block"
+    "keywords": "Volto, React, blocks, grid, container, Plone"
+---
+
+(grid-block)=
+
+# Grid Block
+
+```{versionadded} Volto 17.0.0-alpha.10
+```
+
+Volto comes with a Grid block, building upon the primitive of a container block, it is a block that contains other blocks.
+The Grid block uses a fixed horizontal layout with a configurable maximum items per block, defaulting to four elements.
+
+The Grid block can potentially contain any block type. However, the used blocks might need some styling work to adapt to the horizontal and small widths in the container. For this reason, only these four are included by default: `slate` (text), `image`, `listing` and `teaser`.
+
+## Settings
+
+The Grid has the following settings:
+
+Headline
+: The headline is a heading that appears above the title of the Grid Block.
+
+## Block configuration
+
+The Grid block has these configuration settings:
+
+templates
+: It's a function to provide a list of available templates.
```

Comment:
What is a template?

Reply:
I pushed a more detailed description. 

Now I'm thinking that the TemplateChooser maybe should be part of the particular block, not of the container... I will think about it.

Reply:
@sneridagh I don't have a strong opinion on that. The new description is helpful and helped me realize how the templates could be useful for other kinds of containers.